squad/frontend/templates/squad/_builds_table.html

```diff
@@ -1,62 +1,61 @@
 {% load humanize %}
 {% load squad %}
+<h3> Builds </h3>
 <div class='highlight-row'>
```

Comment:
It would be nice to add a header so the numbers in the table have some meaning.

Reply:
I thought about that but at the moment, but I thought i hold off as I like to replace the current rows with something more condensed and perhaps use some of the table js libraries out there to offer sorting, etc.   Not to mention this has to be styled along with the filters on some other pages.

Reply:
This is not a table (as in <table> tag). It uses class=row from bootstrap to pretend to be table. I don't want <table> here.